### DISCUSSION TITLE
Show number of alpha pixels with opacity <= certain value?

### DISCUSSION THREAD

**Discussion by vin9x** [ImageMagick/ImageMagick#5200] [Help] [answered]
Sadly, I don't how the median or mean value of the verbose info readout could be used to answer this specific question.
Anybody have an idea?

**Comment by snibgo** [comment]
What do you mean by "show"? What output do you want? Text or an image?

Possibly:
```
magick in.ext -alpha extract -threshold 34% out.png
```
That makes out.png with black pixels where the alpha was less than or equal to 34%, otherwise white pixels.

**Reply by vin9x** [reply]
> What do you mean by "show"? What output do you want? Text or an image?
> 
> Possibly:
> 
> ```
> magick in.ext -alpha extract -threshold 34% out.png
> ```
> 
> That makes out.png with black pixels where the alpha was less than or equal to 34%, otherwise white pixels.

Show as text. So output the number of the pixels within the threshold.

**Comment by snibgo** [comment, marked as answer]
The output of my command above is white where the input alpha is above 34%. So the mean of the output is the proportion of the input pixels that have alpha above 34%.
```
magick toes_holed.png -alpha extract -threshold 34% -format %[fx:mean]\n info:

0.370385
```
You can multiply the proportion by the area (in pixels) to get the number of pixels that are above 34%:
```
magick toes_holed.png -alpha extract -threshold 34% -format %[fx:mean*w*h]\n info:

23042
```
